google/earthenterprise
# Upgrade jquery

Upgrading jquery to 3.6.0 for STIG and CVE issues.

Verification Steps:

* Build gee and deploy server
* Verify jquery version in `/opt/google/gehttpd/htdocs/js/jquery.js` is 3.6.0
